packages/core/src/tile/worker/parsers.ts

```diff
@@ -61,8 +61,8 @@ export function parsePolygon(
       }
 
       shape.push(
-        multiply(affineTransform, [point.x!, point.y!, 1])
-          .subset(index([true, true, false]))
+        multiply(affineTransform, [point.x!, point.y!, 0, 1])
```

Comment:
Interesting math here! It could use a one-sentence comment for the benefit of the general reader